server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/RabbitMQWebAdminServerIntegrationTest.java

```diff
@@ -41,32 +48,36 @@
 import org.awaitility.Awaitility;
 import org.awaitility.Duration;
 import org.eclipse.jetty.http.HttpStatus;
-import org.junit.Rule;
-import org.junit.Test;
 import org.junit.experimental.categories.Category;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
 
 import io.restassured.http.ContentType;
 
 @Category(BasicFeature.class)
```

Comment:
does @Category work the same way with junit5?

Reply:
No, I cannot find a resource about it, but tested manually, surefire ignore junit 5 tests marked with `@Category` while it doesn't happen with `@Tag`